dashboard/config/levels/custom/maze/2-3 Bee Functions 2.level

```diff
@@ -1,5 +1,6 @@
 <Karel>
```

Comment:
Is this level used in a real script? I'm wondering if it would be safer to clone it for the purposes of unit testing (and use the clone in `allthethings`), so any changes made to the level don't cause unexpected test failures.

Reply:
The level comes from Courses 1-4, which are not updated regularly. There are also other levels in allthethings that also show up in these courses. I considered cloning the level too, I believe that the cloned level wouldn't automatically have the same translations available. If the cloned level shows English, it defeats the point of the test.

Reply:
Makes sense!